tests/PHPStan/Analyser/AnalyserIntegrationTest.php

```diff
@@ -1307,13 +1307,11 @@ public function testBug10538(): void
 	private function runAnalyse(string $file, ?array $allAnalysedFiles = null): array
 	{
 		$file = $this->getFileHelper()->normalizePath($file);
-		/** @var Analyser $analyser */
```

Comment:
inline `@var` is no longer necessary. even PHPStorm learnt to autocompletes it in the meantime